easysid.cpp

```diff
@@ -255,6 +304,8 @@ EXPORT void run_ms(uint32_t ms)
 int main()
 {
     start_capture("EasySIDTest.wav");
+    set_chip(MOS6581);
+    set_sample_parameters(1022730, SAMPLE_RESAMPLE_INTERPOLATE, 44100);
```

Comment:
The magic number 1022730 should be a named constant with a comment explaining what it is. Same with 44100, as DEFAULT_SAMPLING_RATE, or something.